src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/MethodologyServiceTests.cs

```diff
@@ -515,6 +516,253 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
             }
         }
         
+        [Fact]
+        public async Task GetSummariesTree()
+        {
+            var publication = new Publication
+            {
+                Title = "Publication title"
+            };
+
+            var topic = new Topic
+            {
+                Title = "Topic title",
+                Publications = ListOf(publication)
+            };
+
+            var theme = new Theme
+            {
+                Title = "Theme title",
+                Topics = ListOf(topic)
+            };
+
+            var latestVersions = ListOf(
+                new MethodologyVersion
+                {
+                    Id = Guid.NewGuid(),
+                    MethodologyContent = new MethodologyVersionContent {
+                        Annexes = new List<ContentSection>(),
+                        Content = new List<ContentSection>(),
```

Comment:
I think `new MethodologyVersionContent()` would be sufficient here, since the two properties will be initialised by default. Same comment applies to the other version.